.github/workflows/ci.yaml

```diff
@@ -1,42 +0,0 @@
-name: CI Setup
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    types: [opened, synchronize, reopened]
-jobs:
-  citest:
-    name: Lint and Unittests
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Setup Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.8"
-      - name: Install tox and any other packages
-        run: pip install "tox < 4" #tox has breaking changes in major version 4
-      - name: Run lint checkers
-        run: make lint
-      - name: Run unittests
-        run: make unittests
-      - name: Save PR number to file
-        run: echo ${{ github.event.number }} > PR_NUMBER.txt
-      - name: Archive PR number
-        uses: actions/upload-artifact@v3
-        with:
-          name: PR_NUMBER
-          path: PR_NUMBER.txt
-      - name: Save coverage report
-        run: |
-          report_root="/home/runner/work/prometheus-juju-exporter/prometheus-juju-exporter/tests/unit/report/"
-          cp ${report_root}/coverage.xml cov.xml
-          
-      - name: Archive code coverage results
-        uses: actions/upload-artifact@v3
-        with:
-          name: coverage-report
-          path: ./cov.xml
```

Comment:
This part is required in order to pass PR and coverage information to sonar integration workflow. Could you please add it to the pr.yaml file?

Reply:
It's already there. [lines#48-59](https://github.com/canonical/bootstack-actions/blob/main/.github/workflows/_lint-unit.yaml#L48-L59)

Reply:
You can check that PR_NUMBER and coverage report are stored in artifacts [here](https://github.com/canonical/prometheus-juju-exporter/actions/runs/4308063633).

Reply:
Noted. I was only looking at the main workflow file in bootstack-actions. And since you changed the ci workflow name, sonarqube workflow is not triggered. It is out of scope of this PR, but we need to update sonar.yaml ([line#4](https://github.com/canonical/prometheus-juju-exporter/blob/main/.github/workflows/sonar.yaml#L4)) to align

Reply:
@rgildein I see that you have fix the sonar trigger in charm repo. Maybe do the same for this one also?

Reply:
@agileshaw I did, but I forgot to push it. Thanks.